### ISSUE TITLE
Cluster topology and nodes status interface

### ISSUE THREAD

**Issue by morigs** [GreptimeTeam/dashboard#159]
I'd be cool if Dashboard world be able to display information about cluster status as well as info about nodes, their statuses and connections. Perhaps it should connect directly to meta server in this case?

**Comment by sunng87** [comment]
Yes! There will a dedicated page to display topology and cluster status. The status data of datanodes have been uploaded to meta, now what we need is to find an elegant way to expose them via http, from meta itself or proxy by frontend.

**Comment by morigs** [comment]
Cool, looking forward for the API!
